### ISSUE TITLE
Replace stand-alone keras with tensorflow.keras

### ISSUE THREAD

**Issue by budach** [budach/pysster#1]
Simply replacing all occurrences of `from keras` with `from tensorflow.python.keras` works like a charm - no crashes - BUT: the loss on my data sets doesn't improve over time anymore. Other users observe similar behavior (https://github.com/tensorflow/tensorflow/issues/15026, https://github.com/tensorflow/tensorflow/issues/15831).

I will try again with a future tensorflow version. Having one less dependency is always nice.

**Comment by budach** [comment]
Note: Losses still don't develop with Tensorflow 1.6

**Comment by budach** [comment]
The reason for the differences seems to be that the tensorflow.keras API lags behind stand alone keras API. Not doing it for now.

**Comment by ErmiaAzarkhalili** [comment]
@budach @fchollet what is your final solution for this important bug.
